src/services/external-api-helper.ts

```diff
@@ -147,6 +147,7 @@ export const getFromIMDbAPIV2 = async(imdbId?: string, searchRequest?: SearchReq
       try {
         searchResults = await imdbAPI.search(searchRequest);
       } catch (e) {
+        console.error(e);
```

Comment:
@js-kyle can we ignore the error if it's that one that happens when something isn't found, otherwise log it? Since this change will create unnecessary logspam

Reply:
I'm keen to do it as part of this https://github.com/UniversalMediaServer/api/issues/431 as it needs to be investigated what the error is

Reply:
Ok that sounds good. One last thing is should we also do that checking that the bugs you identified have been fixed for the existing endpoints, or at least make stories for those?

Reply:
Ok I made issues for those things so this is good to go